packages/core/package.json

```diff
@@ -63,6 +71,10 @@
     "@ai-sdk/azure": "^0.0.32",
     "@ai-sdk/mistral": "^0.0.38",
     "@ai-sdk/openai": "^0.0.54",
+    "@latitude-data/jobs": "workspace:^",
+    "@latitude-data/compiler": "workspace:^",
+    "@latitude-data/env": "workspace:^",
+    "@latitude-data/mailers": "workspace:^",
```

Comment:
by declaring dependencies between jit packages as peer dependencies we avoid a "cyclic dependency" warning by pnpm